WordPress/src/main/java/org/wordpress/android/ui/mysite/items/SiteItemsBuilder.kt

```diff
@@ -42,6 +42,7 @@ class SiteItemsBuilder @Inject constructor(
         return jetpackSiteItems + publishSiteItems + lookAndFeelSiteItems + configurationSiteItems + externalSiteItems
     }
 
+    // todo: JetpackFocus - should we be showing the quick start for static posters?
```

Comment:
Since the quick start tasks are there for Reader tab, I feel we should not show the quick start in static posters phase

Reply:
@AjeshRPai - Let's address in a separate PR and have this one focus on the core functionality.